packaging/conf/pbm-conf-reference.yml

```diff
@@ -0,0 +1,112 @@
+######################## PBM Configuration ############################
+
+# This file is a full configuration example documenting all 
+# options in comments. 
+#
+# You can find the full configuration reference here:
+# https://docs.percona.com/percona-backup-mongodb/configuration-options.html
+
+#======================Storage Configuration============================
+
+#storage:
+
+#   Remote backup storage type. Supported types: S3, filesystem, azure
+ 
+
+#---------------------S3 Storage Configuration--------------------------
+#  type:
+#    s3:
+
+# Specify the location and name of the bucket that you have configured on the S3 
+#     region: 
+#     bucket: 
+
+# The data directory to store backups in. 
+# When undefined, backups are saved at the root of the bucket.
+#     prefix:  
+
+# The URL to access the bucket for GCS and MinIO
+#     endpointURL: 
+
+# S3 access credentials.
+#     credentials:
+#       access-key-id: 
+#       secret-access-key: 
+
+# The size of data chinks (in MB) to upload to the bucket.
+#     uploadPartSize: 10
+
+# Data upload configuration
+#     maxUploadParts: 10,000
+
+# Set the storage classes for data objects in the bucket. 
+# If undefined, the default STANDRD object will be used.
+#     storageClass:  
+
+# Allow PBM to upload data to storage with self-issued TLS certificates. 
+# Use it with caution as it might leave a hole for man-in-the-middle attacks. 
+#     insecureSkipTLSVerify:
+
+# Debug level logging configuration for S3 requests.
+#     debugLogLevels: 
+
+# Server-side encryption options.
+#     serverSideEncryption:
+#       sseAlgorithm: aws:kms
+#       kmsKeyID: 
+ 
+# Retry upload configuration options.
+#     retryer:
+#       numMaxRetries: 3
+#       minRetryDelay: 30
+#       maxRetryDelay: 5
+
+#--------------------Filesystem Configuration---------------------------
+#  type:
+#    filesystem:
+
+# The path to backup directory
+#      path: 
+
+
+#--------------------Microsoft Azure Configuration-----------------------
+#  type:
+#    azure:
+
+# Azure storage account and container name
+#      account: 
+#      container: 
+
+# Where to store data in the container
+#      prefix: 
+
+# Specify the access key
+#      credentials:
+#        key: 
+
+#====================Point-in-Time Recovery Configuration==================
+
+#pitr:
+#  enabled: false
+
+#Adjust the size op oplog chunks (in minutes).
+#  oplogSpanMin: 10 
+
+#Set a compression method and level
```

Comment:
pls ensure space character between `#` and the next letter character for consistency with the whole example